wallets/provider-ledger/package.json

```diff
@@ -25,6 +25,9 @@
     "@ledgerhq/hw-app-eth": "^6.36.0",
     "@ledgerhq/hw-app-solana": "^7.1.6",
     "@ledgerhq/hw-transport-webhid": "^6.28.6",
+    "@ledgerhq/types-cryptoassets": "^7.11.0",
+    "@types/w3c-web-hid": "^1.0.2",
+    "@ledgerhq/types-devices": "^6.24.0",
```

Comment:
I guess not, since we are generating `.d.ts` as well and if we don't put these in `dependencies` it won't be installed on host machine. And package user will get an error, like what we get from @ledegerhq package it we added their dependencies manually.